executor/write_test.go

```diff
@@ -1103,13 +1103,13 @@ func (s *testSuite) TestLoadData(c *C) {
 	// fields and lines are default, InsertData returns data is nil
 	tests := []testCase{
 		// data1 = nil, data2 != nil
-		{nil, []byte("\n"), []string{"1|0||0"}, nil},
-		{nil, []byte("\t\n"), []string{"2|0||0"}, nil},
+		{nil, []byte("\n"), []string{"1|<nil>|<nil>|<nil>"}, nil},
```

Comment:
`\n` is not null, `\N` is null

Reply:
It deals with the number of columns less than it in the table.